meta-mel/recipes-core/os-release/os-release.bbappend

```diff
@@ -0,0 +1,13 @@
+OS_RELEASE_FIELDS = "PRETTY_NAME NAME VERSION_ID VERSION VERSION_CODENAME ID HOME_URL SUPPORT_URL BUG_REPORT_URL BUILD_ID"
+OS_RELEASE_UNQUOTED_FIELDS = "ID VERSION_ID VARIANT_ID"
+
+ID = "${DISTRO}"
+NAME = "${OS_NAME}"
+VERSION = "${DISTRO_VERSION}${@' (%s)' % DISTRO_CODENAME if 'DISTRO_CODENAME' in d else ''} ${YOCTO_RELEASE}"
```

Comment:
Hmm, it looks like DISTRO_CODENAME actually holds the yocto release for some distros, i.e. the imx and poky distro. In our case, it would probably hold our private name for the release, but if that won't be public anyway, we might just alter our DISTRO_CODENAME to hold the yocto release. It's debatable, but I think we *should* set DISTRO_CODENAME regardless, to either our name or the upstream one. And if we're keeping both, we may want to do the same thing that's done for DISTRO_CODENAME here, rather than putting the parenthesis in the variable contents.

Reply:
We could potentially either set `DISTRO_CODENAME` to `dunfell` or `our-codename (dunfell)`.

Reply:
@kergoth I set the DISTRO_CODENAME in mel.conf. see new commit